plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -141,8 +141,9 @@ Setting a color to black (#000000) will make it invisible.""")
     public static Map<Block, Integer> BLOCK_COLORS = new LinkedHashMap<>() {{
         put(Blocks.ACACIA_BUTTON, 0x000000);
         put(Blocks.ACACIA_DOOR, 0xA85F3D);
-        put(Blocks.ACACIA_FENCE, 0xA85A32);
         put(Blocks.ACACIA_FENCE_GATE, 0xA85A32);
+        put(Blocks.ACACIA_FENCE, 0xA85A32);
```

Comment:
Looks like your sorting algorithm is placing `_` character above null characters. The C standard places underscores after null chars and capitals, but only before lowercase characters. So, `ACACIA_FENCE` should be above `ACACIA_FENCE_GATE`

Example with random words in a text file, shown as is, then using UTF8 standard, then C standard.
![image](https://user-images.githubusercontent.com/332527/180653515-22a36f6c-b81b-468f-ad4f-26bc8be7a094.png)

I like the C standard because it looks cleanest to me :3